components/helpers/fire-event.js

```diff
@@ -0,0 +1,20 @@
+define('fire-event', [], () =>
+    /**
+     * Cross browser method to fire events.
+     *
+     * @param {HTMLElement} el - target DOM element to fire the event at.
+     * @param {string} eventName - the name of the event to fire.
+     * @returns {undefined}
+     */
+    function fireEvent(el, eventName) {
```

Comment:
This all looks a little squashed up, I'd break up with some lines, just my opinion though